pages/login.vue

```diff
@@ -1,48 +1,137 @@
+                class="block text-sm font-medium leading-6 text-gray-900"
+              >Username</label>
+              <input
+                id="username"
+                type="text"
+                name="username"
+                autocomplete="username"
+                autofocus
+                required
+                class="block w-full rounded-md border-0 p-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+              >
+            </div>
+            <div class="sm:col-span-3 flex flex-col gap-1">
+              <label
+                for="password"
+                class="block text-sm font-medium leading-6 text-gray-900"
+              >Password</label>
+              <input
+                id="password"
+                type="password"
+                name="password"
+                autocomplete="password"
+                required
+                class="block w-full rounded-md border-0 p-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+              >
+            </div>
+            <button
+              type="submit"
+              class="rounded-md bg-indigo-600 px-3 py-2 mt-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+            >
+              Log in
+            </button>
+          </form>
         </div>
-        <div class="sm:col-span-3 flex flex-col gap-1">
-          <label
-            for="password"
-            class="block text-sm font-medium leading-6 text-gray-900"
-          >Password</label>
-          <input
-            id="password"
-            type="password"
-            name="password"
-            autocomplete="password"
-            required
-            class="block w-full rounded-md border-0 p-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
-          >
-        </div>
-        <button
-          type="submit"
-          class="rounded-md bg-indigo-600 px-3 py-2 mt-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
-        >
-          Log in
-        </button>
-      </form>
+      </div>
+      <div
+        ref="separator"
+        class="w-1 bg-gray-300 cursor-ew-resize"
+      />
+      <div
+        ref="resizeGameElement"
+        class="flex min-w-[300px] w-1/2 overflow-auto"
+      >
+        <GameScreen />
+      </div>
     </div>
   </div>
 </template>
+
+<script lang="ts">
+import { defineComponent, ref, onMounted, onUnmounted } from "vue";
+
+export default defineComponent({
```

Comment:
It's not ideal that we've copied this separator-related code. Maintaining two copies of it will be a chore. It would be better if we can:
- extract it to a component, call it a `SeparatorContainer`
- use the component like this
```vue
<SeparatorContainer>
  <LoginForm />
  <GameScreen />
</SeparatorContainer>
```

Another option is to remove the separator from the login screen in this PR – I am not sure if we need it on the login screen yet.

Reply:
Thank you for the feedback!
Will get back to you once I make the changes

Reply:
@yurijmikhalevich , I have removed the resizing logic from the login page

Reply:
@arilloid, thank you for the update! Looks good. I have left two small comments.